src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -181,7 +181,19 @@ namespace {{asUpperCamelCase label}} {
         static constexpr bool MustUseTimedWrite() { return {{mustUseTimedWrite}}; }
     };
 } // namespace {{asUpperCamelCase label}}
+
 {{#last}}
+namespace AttributeList {
+    struct TypeInfo {
+        using Type = DataModel::List<AttributeId>;
+        using DecodableType = DataModel::DecodableList<AttributeId>;
+        using DecodableArgType = const DataModel::DecodableList<AttributeId> &;
+        static constexpr ClusterId GetClusterId() { return Clusters::{{asUpperCamelCase parent.name}}::Id; }
+        static constexpr AttributeId GetAttributeId() { return Attributes::AttributeList::Id; }
+        static constexpr bool MustUseTimedWrite() { return false; }
+    };
+}
+
```

Comment:
Ah. We need that because this file is not using `ClusterHelper` right ? That's annoying.

I guess this is the same cause for manually adding it into the other files that are generated into `app-common`

Reply:
Yep.  But again, adding the global `<attribute>` fixed this part, so this change is gone.